src/ansys/units/quantity.py

```diff
@@ -51,50 +58,26 @@ def __new__(
         ):
             raise QuantityError.EXCESSIVE_PARAMETERS()
 
-        if value == None and not copy_from:
-            raise QuantityError.MISSING_REQUIREMENT()
-
         if copy_from:
             if value:
+                self.value = value
```

Comment:
What is the difference between `self.value` and `self._value`?

I think I need some of the logic clarified here. We can catch up today.

N.b.: `units = copy_from.units` is done in both branches here.

Reply:
I believe it should be `self._value` where it would be interacted with as a protected variable for better encapsulation. @seanpearsonuk please let me know when you would like to catch up.